miniapps/shifted_methods/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
The copyright dates in all relevant files should probably be incremented to 2021.

Reply:
I merged `master` and the [`copyright-check`](https://github.com/mfem/mfem/pull/2040/checks?check_run_id=1874077067) test shows that the following files need to be updated:

```
miniapps/shifted_methods/CMakeLists.txt
miniapps/shifted_methods/makefile
```